src/copy.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+const copyFile = (fromPath, toPath) => {
+  if (!fs.existsSync(fromPath)) {
+    console.error('Source file does not exist!');
+
+    return;
+  }
+
+  fs.copyFileSync(fromPath, toPath);
```

Comment:
Recommend you to wrap this method with `try...catch` and catch an error that can occur.